### ISSUE TITLE
On the fly auto import doesn't work on 2022.3

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#9418]
## Environment

* **IntelliJ Rust plugin version:** 0.4.180.SNAPSHOT-223-dev
* **Rust toolchain version:** 1.64.0 (a55dd71d5 2022-09-19) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2022.3 EAP Ultimate Edition (IU-223.4884.79)
* **Operating system:** macOS 12.5.1
* **Macro expansion:** enabled
* **Additional experimental features:** org.rust.cargo.emulate.terminal, org.rust.cargo.evaluate.build.scripts

## Steps to reproduce
```rust
fn main() {
    func();
}
mod inner {
    pub fn func() {}
}
```

**Comment by dima74** [comment]
This will probably be fixed in next EAP, so lets wait for it (https://youtrack.jetbrains.com/issue/IDEA-301898)

**Comment by dima74** [comment]
Fixed in #9585
